src/dev-utils/up.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -ex
+
+# This script assume code repo and config repo are clean
+
+CODE_BASE_DIR="/mnt/dev/dlts"
+CODE_UPSTREAM_REMOTE="upstream"
+CONFIG_DIR="$HOME/dev/Deployment"
+
+if [ $# != 2 ] ; then
+    echo Usage: $0 cluster_name version branch
+    exit 1
+fi
+
+cluster_name=$1
+version=$2
+
+#(cd $CONFIG_DIR ; git fetch origin ; git merge origin/master)
+
+cd $CODE_BASE_DIR/$cluster_name
+
+git checkout . # make code path clean
+
+git fetch --all ; git checkout $CODE_UPSTREAM_REMOTE/$version -b $version
+
+cd src/ClusterBootstrap
+
+./ctl.py restorefromdir $CONFIG_DIR/$cluster_name
+
+# start to deploy
+
+for i in `echo restfulapi storagemanager repairmanager dashboard`
+do
+    ./ctl.py svc configupdate $i
+done
+
+./ctl.py kubectl label node `grep infra01: status.yaml  | sed 's/ *:*//g'` job-insighter=active
```

Comment:
infra node name does not necessarily contain `infra01`, but we can refine later.

Reply:
ok